### DISCUSSION TITLE
How do contributers make their forks FEBuilder compatible?

### DISCUSSION THREAD

**Discussion by JesterWizard** [MokhaLeee/fe8u-cskillsys-kernel#350] [General]
I've seen in your releases page you have a little patches zip with a bunch of files that slot into FEBuilder to allow users to assign skills. I've never seen this created myself. Presumably this is created automatically, so what tool do you use to do it?

**Comment by MokhaLeee** [comment]
They are handwritten

**Reply by JesterWizard** [reply]
Darn, do you have any instructions I could follow to do the same?

**Reply by MokhaLeee** [reply]
you can see this: https://laqieer.github.io/dw.ngmansion.xyz/wiki/en/guide/febuildergba/the_config_data_structure_of_febuildergba_en.html
